modules_core/org.openbravo.client.application/src/org/openbravo/client/application/window/FormInitializationComponent.java

```diff
@@ -1677,11 +1677,16 @@ private void manageUpdatedValuesForCallout(Map<String, JSONObject> columnValues,
               // input. We assume it is a hidden input, which are used in places like
               // selectors
               Object el = calloutInformationProvider.getCurrentElementValue(element);
-              if (el != null && (!calloutInformationProvider.isComboData(element))) {
-                hiddenInputs.put(name, el);
-                // We set the hidden fields in the request, so that subsequent callouts
-                // can use them
-                request.setRequestParameter(name, el.toString());
+              if (el != null) {
+                if (calloutInformationProvider.isComboData(element)) {
```

Comment:
```
               if (calloutInformationProvider.isComboData(element)) {
                  // In this case, we ignore the value, as a hidden input cannot be an array
                  // of elements
                } else {
                  hiddenInputs.put(name, el);
                  // We set the hidden fields in the request, so that subsequent callouts
                  // can use them
                  request.setRequestParameter(name, el.toString());
                }
```

No entiendo porque te saltas el caso del true en el if. No seria mejor que quedara en una sola condicion negando el if?

```
                if (!calloutInformationProvider.isComboData(element)) {
                  hiddenInputs.put(name, el);
                  // We set the hidden fields in the request, so that subsequent callouts
                  // can use them
                  request.setRequestParameter(name, el.toString());
                }
```

Reply:
Lo revisamos con @valeg-etendo , decidimos dejarlo así y evitarnos que tenga un uso que desconozcamos.